x-pack/plugins/search_indices/public/components/indices/delete_index_modal.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { Fragment, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiConfirmModal } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useDeleteIndex } from '../../hooks/api/use_delete_index';
+interface DeleteIndexModelProps {
+  onCancel: () => void;
+  indexName: string;
+  navigateToIndexListPage: () => void;
+}
+export const DeleteIndexModal: React.FC<DeleteIndexModelProps> = ({
+  onCancel,
+  indexName,
+  navigateToIndexListPage,
+}) => {
+  const { mutate, isLoading, isSuccess } = useDeleteIndex(indexName);
+  useEffect(() => {
+    if (isSuccess) {
+      navigateToIndexListPage();
+    }
+  }, [navigateToIndexListPage, isSuccess]);
+  return (
+    <EuiConfirmModal
+      data-test-subj="deleteIndexActionModal"
+      title={i18n.translate(
+        'xpack.searchIndices.indexActionsMenu.deleteIndex.confirmModal.modalTitle',
+        {
+          defaultMessage: 'Delete {index}',
```

Comment:
I think the modal title should be more general and simply read 'Delete index'—especially since we are also including the index name in the body of the modal.

Reply:
updated in 5dde7ca7226ddcf1e89c639b17d6b16ebc34eba7